data.sql

```diff
@@ -19,4 +19,35 @@ values ('Angemon', DATE '2005-06-12', 1, true, -45);
 INSERT INTO animals(name, date_of_birth, escape_attempts, neutered, weight_kg)
 values ('Boarmon', DATE '2005-06-07', 7, true, 20.4);
 INSERT INTO animals(name, date_of_birth, escape_attempts, neutered, weight_kg)
-values ('Blossom', DATE '1998-10-13', 3, true, 17);
\ No newline at end of file
+values ('Blossom', DATE '1998-10-13', 3, true, 17);
```

Comment:
- [ ] You are missing to insert the data in the species table
- [ ] Let us keep plural form  our query solution (`INSERT, INTO and VALUES`)